deposit-app/src/test/java/edu/unc/lib/boxc/deposit/fcrepo4/IngestContentObjectsJobTest.java

```diff
@@ -322,29 +331,37 @@ public void ingestWorkObjectTest() throws Exception {
     }
 
     /**
-     * Test that the deposit fails if a file object is specified but not given a
-     * staging location
+     * Test that ingest will work if a file object is specified with no original resource,
+     * assuming it has streaming properties and passed validation
      */
     @Test
-    public void ingestWorkWithFileWithoutLocation() throws Exception {
-        Assertions.assertThrows(JobFailedException.class, () -> {
-            PID workPid = makePid(RepositoryPathConstants.CONTENT_BASE);
-            WorkObject work = mock(WorkObject.class);
-            Bag workBag = setupWork(workPid, work);
+    public void ingestWorkWithFileWithoutOriginalFileAndStagingLocation() {
+        PID workPid = makePid(RepositoryPathConstants.CONTENT_BASE);
+        WorkObject work = mock(WorkObject.class);
+        Bag workBag = setupWork(workPid, work);
+
+        PID filePid = makePid(RepositoryPathConstants.CONTENT_BASE);
 
-            PID filePid = makePid(RepositoryPathConstants.CONTENT_BASE);
+        Resource fileResc = model.createResource(filePid.getRepositoryPath());
+        fileResc.addProperty(RDF.type, Cdr.FileObject);
+        fileResc.addProperty(CdrDeposit.mimetype, "text/plain");
+        fileResc.addProperty(Cdr.streamingHost, DURACLOUD);
+        fileResc.addProperty(Cdr.streamingFolder, CLOSED);
+        fileResc.addProperty(Cdr.streamingFile, "banjo-playlist.m3u8");
+        workBag.add(fileResc);
 
-            Resource fileResc = model.createResource(filePid.getRepositoryPath());
-            fileResc.addProperty(RDF.type, Cdr.FileObject);
-            fileResc.addProperty(CdrDeposit.mimetype, "text/plain");
-            workBag.add(fileResc);
+        job.closeModel();
 
-            job.closeModel();
+        when(repoObjLoader.getWorkObject(eq(workPid))).thenReturn(work);
+        when(repoObjFactory.createFileObject(eq(filePid), any())).thenReturn(streamingFileObj);
+        when(streamingFileObj.getPid()).thenReturn(filePid);
 
-            when(repoObjLoader.getWorkObject(eq(workPid))).thenReturn(work);
+        job.run();
 
-            job.run();
-        });
+        verify(repoObjFactory).createWorkObject(eq(workPid), any(Model.class));
+        verify(destinationObj).addMember(eq(work));
+
+        verify(jobStatusFactory, times(2)).incrCompletion(eq(jobUUID), eq(1));
     }
 
```

Comment:
I think we should add a test for a file object that has both streaming properties and an original resource. I don't have an opinion whether that gets added to the unit or integration test